experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml

```diff
@@ -0,0 +1,208 @@
+- hosts: localhost
+  connection: local
+
+  vars:
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_pvc: "{{ lookup('env','APP_PVC') }}"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    c_experiment: "openebs-pool-disk-loss"
+    ramp_time: "{{ lookup('env','RAMP_TIME') }}"
+    data_persistence: "{{ lookup('env','DATA_PERSISTENCE') }}"
+    liveness_label: "{{ lookup('env','LIVENESS_APP_LABEL') }}"
+    liveness_namespace: "{{ lookup('env','LIVENESS_APP_NAMESPACE') }}"
+    openebs_ns: "{{ lookup('env','OPENEBS_NAMESPACE') }}"
+    cloud_platform: "{{ lookup('env','CLOUD_PLATFORM') }}"
+    c_ns: "{{ lookup('env','CHAOS_NAMESPACE') }}"
+    disk_name: "{{ lookup('env','DISK_NAME') }}"
+    device_name: "{{ lookup('env', 'DEVICE_NAME') }}"
+    node_name: "{{ lookup('env','NODE_NAME') }}"
+    project_id: "{{ lookup('env','PROJECT_ID') }}"
+    zone_name: "{{ lookup('env','ZONE_NAME') }}"
+    auxiliary_appinfo: "{{ lookup('env','AUXILIARY_APPINFO') }}"
+
+  vars_files:
+    - /mnt/parameters.yml
+    - /experiments/openebs/openebs_components.yml
+  
+  tasks:
+    - block:
+
+      - include_tasks: /utils/runtime/create_testname.yml  
+
+      - include_tasks: openebs_pool_disk_loss_ansible_prerequisites.yml
+
+      # GENERATE EXPERIMENT RESULT NAME AND CREATE CHAOS RESULT RESOURCE
+      - name: Construct chaos result name (experiment_name)
+        set_fact:
+          c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
+        when: lookup('env','CHAOSENGINE')
+
+      - include_tasks: /utils/runtime/update_chaos_result_resource.yml
+        vars:
+          status: 'SOT'
+          namespace: "{{ a_ns }}"
+
+      # DISPLAY APP INFORMATION
+      - name: Display the app information passed via the test job
+        debug:
+          msg:
+            - "The application info is as follows:"
+            - "Namespace    : {{ a_ns }}"
+            - "Label        : {{ a_label }}"
+            - "PVC        : {{ a_pvc }}"
+
+      - block: 
+          # Gcloud authentication
+          - name: Gcloud authentication
+            include_tasks: "/utils/cloud/gcp/gcloud_configure.yml"
+            when: "cloud_platform == 'GCP'"
+          
+          # AWS authentication
+          - name: AWS authentication
+            include_tasks: "/utils/cloud/aws/aws_configure.yml"
+            when: "cloud_platform == 'AWS'"
+
+      ## PRE-CHAOS DISK LIVENESS CHECK
+      - block:
+        - name: Verify that the disk is connected to node (pre)
```

Comment:
In some places, you put 2 spaces ( line 68) and at some places you put 4 spaces ( line-57) for the block items. Can you make it uniform [ 4 spaces ] at all places?

Reply:
It's already 2 space. PTAL

Reply:
yeah Its 2 space. I am asking to make it four

Reply:
As discussed we are going to make 2 spaces in all the ansible playbook